### PR TITLE
docs: Desaturate the docs of all the yaml

### PR DESCRIPTION
**Pull request by Callisto13**
There has been feedback (eg https://github.com/weaveworks/weave-gitops-enterprise/issues/524) that tons of yaml and output etc all over the docs makes it challenging to read. This commit puts most of the biggest chunks into expandable `details` tags. The result is a cleaner, faster read. I am sure I have missed a couple, but this is fine for now.

I have done some minor changes to text so that it works with the collapsed views, but otherwise this is just moving around things which already exist.

### CODE REVIEW

File: website/docs/cluster-management/managing-existing-clusters.mdx [hunk 1]
```diff
@@ -36,146 +36,154 @@ kubectl create secret generic demo-01-kubeconfig \
...
+	apiVersion: v1
+	kind: ServiceAccount
+	metadata:
+		name: demo-01
+		namespace: default
+	```
+
+1. Add RBAC permissions for the service account
+
+	<details><summary>Expand to see role manifests</summary>
```
Comment (enekofb) [inline website/docs/cluster-management/managing-existing-clusters.mdx:49]: Do you see the same rendering issue? 

Screenshot 2023-02-21 at 10 32 56

File: website/docs/cluster-management/managing-existing-clusters.mdx [hunk 2]
```diff
@@ -36,146 +36,154 @@ kubectl create secret generic demo-01-kubeconfig \
...
+
+	```sh
+	TOKEN=$(kubectl get secret demo-01-token-gqz7p -o jsonpath={.data.token} | base64 -d)
+	```
+
+1. Create a kubeconfig secret
+
+	We'll use a helper script to generate the kubeconfig, save this into `static-kubeconfig.sh`:
+
+	<details><summary>Expand to see script</summary>
```
Comment (enekofb) [inline website/docs/cluster-management/managing-existing-clusters.mdx:107]: This looks good 👍 
Screenshot 2023-02-21 at 10 34 59

File: website/docs/cluster-management/managing-existing-clusters.mdx [hunk 3]
```diff
@@ -228,7 +236,10 @@ Once flux has reconciled the cluster you can inspect your flux resources via the
 
 To test a kubeconfig secret has been correctly setup apply the following manifest and check the logs after the job completes:
 
+<details><summary>Exoand to see manifest</summary>
```
Comment (enekofb) [inline website/docs/cluster-management/managing-existing-clusters.mdx:239]:
```suggestion
<details><summary>Expand to see manifest</summary>
```

File: website/docs/cluster-management/managing-existing-clusters.mdx [hunk 4]
```diff
@@ -261,6 +272,8 @@ spec:
             optional: false
 ```
 
+</details>
+
 In the manifest above `demo-01-kubeconfig`is the name of the secret that contains the kubeconfig for the remote cluster.
```
Comment (enekofb) [inline website/docs/cluster-management/managing-existing-clusters.mdx:277]:
```suggestion
In the manifest above `demo-01-kubeconfig` is the name of the secret that contains the kubeconfig for the remote cluster.
```

File: website/docs/cluster-management/managing-existing-clusters.mdx (hunk 1 above)
Comment (Callisto13) [reply website/docs/cluster-management/managing-existing-clusters.mdx:49]: 👀 will check it out

Comment (Callisto13) [reply website/docs/cluster-management/managing-existing-clusters.mdx:49]: ah yeh forgot to leave a newline

File: website/docs/configuration/user-permissions.mdx [hunk 5]
```diff
@@ -163,6 +167,8 @@ In order to view canaries in a remote cluster from the management cluster, you n
 
 For example, applying the following manifest on remote clusters, ensures that the `wego-admin` user will be able to view canary information from within the Weave GitOps Enterprise UI on the management cluster:
 
+<details><summary>Expand to see example service user RBAC</summary>
```
Comment (enekofb) [inline website/docs/configuration/user-permissions.mdx:170]:
```suggestion
<details><summary>Expand to see example of remote cluster canary reader RBAC</summary>
```

Comment (Callisto13) [reply website/docs/configuration/user-permissions.mdx:170]: 🍝

File: website/docs/guides/setting-up-dex.md [hunk 6]
```diff
@@ -60,7 +60,9 @@ kubectl create secret generic github-client \
 As we did before, we can use `HelmRepository` and `HelmRelease` objects to let
 Flux deploy everything.
 
-```yaml
+<details><summary>Expand to see resource manifests</summary>
+
+```yaml {43,52,66-74,88-89}
```
Comment (enekofb) [inline website/docs/guides/setting-up-dex.md:65]: what are your thoughts behind highlighting these lines?

File: website/docs/installation/weave-gitops-enterprise/airgap.mdx [hunk 7]
```diff
@@ -461,12 +490,15 @@ enterprise.Now you are ready to install WGE from your private registry.
...
-1. Adjust Helm Releases `spec.chart.sourceRef` to tell Flux to pull helm charts from your helm repo.
+1. Adjust Helm Releases `spec.chart.spec.sourceRef` to tell Flux to pull helm charts from your helm repo.
 2. Adjust Helm Releases `spec.values` to use the container images from your private registry.
 
 An example of how it would look like for Weave Gitops Enterprise is shown below.
 
-```yaml
+<details><summary>Expand to view example WGE manifest</summary>
+
+```yaml title="weave-gitops-enterprise.yaml" {21-24,32}
```
Comment (enekofb) [inline website/docs/installation/weave-gitops-enterprise/airgap.mdx:500]: 🏅

File: website/docs/guides/setting-up-dex.md (hunk 6 above)
Comment (Callisto13) [reply website/docs/guides/setting-up-dex.md:65]: oh actually i meant to remove that highlighting

File: website/docs/policy/policy-configuration.mdx [hunk 8]
```diff
@@ -254,26 +260,25 @@ Deploying `app-a` in another namespace other than `flux-system` won't be affecte
...
-    config:
-      weave.policies.policy-1:
-        params:
-          replica_count: 3
-      weave.policies.policy-2:
-        params:
-          run_as_root: true
-  ```
+	:::note
+	You can use one or more policies as the following example
```
Comment (enekofb) [inline website/docs/policy/policy-configuration.mdx:264]: we could adjust this 

Screenshot 2023-02-21 at 12 43 49

File: website/docs/guides/delivery.mdx [hunk 9]
```diff
@@ -82,7 +82,18 @@ directory.
...
 
+Now let's add the resources for Linkerd components under the `./linkerd`
+directory. These are:
+- A `Namespace` resource to control where the components are installed
+- A `HelmRepository` resource to make the Linkerd helm repo available on the
+	cluster
+- A `HelmRelease` resource to install the latest version of Linkerd from that
+	`HelmRepository`
+
+<details><summary>Expand to see the linkerd resource manifests</summary>
```
Comment (enekofb) [inline website/docs/guides/delivery.mdx:93]:
```suggestion
<details><summary>Expand to see Linkerd manifests</summary>
```

**Review by enekofb — COMMENTED** [review]
Reviewed expandable sections. Most of them looks fine ... just a couple requires adjustment.  Other small suggestions added. 

Thanks @Callisto13 🙏

**Comment by Callisto13** [comment]
🙏 thanks for the review @enekofb!

Changes added and pushed

**Review by enekofb — APPROVED** [review]
🚀

**Pull request merged**